src/main/java/org/example/Twitter.java

```diff
@@ -0,0 +1,461 @@
+package org.example;
+
+import java.io.*;
+import java.util.*;
+
+import java.util.Scanner;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.json.simple.parser.*;
+
+class Tweet {
+    String content;
+    int userId;
+    int tweetId;
+
+    ArrayList<String> hashtags;
+
+    public String getContent() {
+        return content;
+    }
+
+    public void setContent(String content) {
+        this.content = content;
+    }
+
+    public int getUserId() {
+        return userId;
+    }
+
+    public void setUserId(int userId) {
+        this.userId = userId;
+    }
+
+    public int getTweetId() {
+        return tweetId;
+    }
+
+    public void setTweetId(int tweetId) {
+        this.tweetId = tweetId;
+    }
+
+    public ArrayList<String> getHashtags() {
+        return hashtags;
+    }
+
+    public void setHashtags(ArrayList<String> hashtags) {
+        this.hashtags = hashtags;
+    }
+
+}
+
+class User {
+    int userId;
+    String name = "";
+    String handle;
+    String username;
+    String password;
+    ArrayList<Integer> tweets = new ArrayList<Integer>();
+    ArrayList<Integer> followers = new ArrayList<Integer>();
+    ArrayList<Integer> following = new ArrayList<Integer>();
+
+    public int getUserId() {
+        return userId;
+    }
+
+    public void setUserId(int userId) {
+        this.userId = userId;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getHandle() {
+        return handle;
+    }
+
+    public void setHandle(String handle) {
+        this.handle = handle;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public ArrayList<Integer> getTweets() {
+        return tweets;
+    }
+
+    public void setTweets(ArrayList<Integer> tweets) {
+        this.tweets = tweets;
+    }
+
+    public ArrayList<Integer> getFollowers() {
+        return followers;
+    }
+
+    public void setFollowers(ArrayList<Integer> followers) {
+        this.followers = followers;
+    }
+
+    public ArrayList<Integer> getFollowing() {
+        return following;
+    }
+
+    public void setFollowing(ArrayList<Integer> following) {
+        this.following = following;
+    }
+
+} 
+
+
+public class Twitter
+{
+    public static int currentUserId = -1;
+    public static int tweetCounter = 0;
+    public static LinkedHashMap<String, Tweet> tweetDict = new LinkedHashMap<String, Tweet>();
+    public static LinkedHashMap<String, User> userDict = new LinkedHashMap<String, User>();
```

Comment:
can be compressed to list for memory efficiency given unique integer identifiers

Reply:
Using usernames as identifiers, this code needs a hashmap.